### PR TITLE
Product pricebooks

### PR DESCRIPTION
**Pull request by vijijustin**
Please review the products ,price books & pricebook_items API documentation!

### CODE REVIEW

File: api/planmill1_5.raml [hunk 1]
```diff
@@ -1253,7 +1277,86 @@ resourceTypes:
...
+          description: Filters products for a specified pricebook id
+          type: integer
+          required: false
+          example: id={pricebook id}
+  type:
+    collection:
+       exampleCollectionItemArray: !include pricebook_array1_5.sample
+       exampleCollectionItem: !include pricebook1_5.sample
+  /meta:
+      description: Get meta information for productss. Includes resource's fields names and type and related filters
```
Comment (corvuscornix) [inline api/planmill1_5.raml]: Fix typo in productss and add punctuation mark after last sentence.

File: api/planmill1_5.raml [hunk 2]
```diff
@@ -1253,7 +1277,86 @@ resourceTypes:
...
+    type:
+     collectionItem:
+        exampleCollectionItem: !include pricebook1_5.sample
+    /items:
+      type:
+        readOnlyCollection:
+          schemaName: !include pricebookitem1_5.schema
+          exampleCollectionItemArray: !include pricebook_item_array1_5.sample
+      /meta:
+          description: Get meta information for pricebook items. Includes resource's fields names and type and related filters
```
Comment (corvuscornix) [inline api/planmill1_5.raml]: Add punctuation mark after last sentence.

File: api/planmill1_5.raml [hunk 3]
```diff
@@ -1253,7 +1277,86 @@ resourceTypes:
...
+          get:
+          type:
+            readOnlyCollectionItem:
+              exampleCollectionItem: !include pricebookitems_meta1_5.sample
+      /{item_id}:
+        type:
+          putGetDeleteCollectionItem:
+            exampleCollectionItem: !include pricebook_item1_5.sample
+        /meta:
+              description: Get meta information for pricebook specific item. Includes resource's fields names and type and related filters
```
Comment (corvuscornix) [inline api/planmill1_5.raml]: Add punctuation mark after last sentence.

Comment (vijijustin) [reply api/planmill1_5.raml]: Done all review changes

**Pull request merged**